### PR TITLE
created initial express server

### LINKED ISSUE #3: Initial Commit for server
Initial Commit for server

### PR DESCRIPTION
**Pull request by avishkaindula**

### CODE REVIEW

**Review by AmreetKumarkhuntia — APPROVED** [review]

**Comment by AmreetKumarkhuntia** [comment]
@avishkaindula can you add a readme file for the installation procedure so that everyone can follow that installation guide to run the server on their local devices?

File: server/index.js
Comment (AmreetKumarkhuntia) [inline server/index.js:1]: Can you move the index and routes file to another folder src?

Comment (avishkaindula) [reply server/index.js:1]: Sure. Will do @AmreetKumarkhuntia

**Comment by avishkaindula** [comment]
@AmreetKumarkhuntia I've created the installation guide on README.md and moved the index.js and routes folder into src directoroy.
Thank you.

**Review by AmreetKumarkhuntia — CHANGES_REQUESTED** [review]
Add the Readme file inside the server folder

**Comment by AmreetKumarkhuntia** [comment]
#3

**Comment by AmreetKumarkhuntia** [comment]
@avishkaindula please add a commit for the required changes.

**Pull request merged**